Update the doc about a known issue (#2565)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/export-and-import-search-settings.md

|^  <br/> |caret  <br/> |
|\'  <br/> |escape sequence  <br/> |
|\"  <br/> |escape sequence  <br/> |

## Known issue
<a name="__toc351540666"> </a>

> When you import search configuration files in tenant admin search settings page, you may encounter an issue that the **Search Config List** could not display properly and error is "File not found".

Reviewer: ```suggestion
When you import search configuration files into the tenant admin search settings page, you might encounter an issue that the **Search Config List** could not display properly and you might receive a "File not found" error.
```

Author: Resolved.

> This issue only blocks UI from displaying the list of search configuration file you imported and would not break the **Import** functionality. Your search configuration will be imported properly.

Reviewer: ```suggestion
This issue only blocks the UI from displaying the list of search configuration files you imported and would not break the **Import** functionality. Your search configuration will be imported properly.
```

Author: Resolved.

> Since you are unable to check that status of search configuration file you imported from UI, you could choose alternative way to access the list, like [SharePoint CSOM API](https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/complete-basic-operations-using-sharepoint-client-library-code) or [SharePoint Online REST API](https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/complete-basic-operations-using-sharepoint-rest-endpoints).

Reviewer: ```suggestion
Since you are unable to check the status of the search configuration file you imported from the UI, you could choose an alternative way to access the list, like [SharePoint CSOM API](https://docs.microsoft.com/harepoint/dev/sp-add-ins/complete-basic-operations-using-sharepoint-client-library-code) or [SharePoint Online REST API](https://docs.microsoft.com/sharepoint/dev/sp-add-ins/complete-basic-operations-using-sharepoint-rest-endpoints).
```

Author: Resolved.
